tests/HelpMeCarry.tex

```diff
@@ -1,77 +1,137 @@
+  \item \textbf{Start:} The robot starts waiting inside the arena.
+  \item \textbf{Car:} The car is any landmark chosen (but \emph{not} announced) beforehand outside the arena. Several bags with groceries are located where the car is parked. 
   \item \textbf{Doors:} All doors in the apartment are open.
   \item \textbf{Operator:} A \quotes{professional} operator is selected by the TC to act as the operator of the robot. 
-  \item \textbf{Other people:} There are no restrictions on other people walking by or standing around throughout the complete task. 
+  \item \textbf{Uncontrolled environment:} There are no restrictions on other people walking by or standing around throughout the complete task. 
 \end{enumerate}
 
 \subsection{Task}
 \begin{enumerate}
-\item \textbf{Start:} The robot is waiting somewhere in the apartment. The operator approaches the robot and tells it to follow. E.g. ``Robot, follow me''
-\item \textbf{Following:} The robot starts following the operator, who guides the robot to the car containing the groceries. 
-\item \textbf{Arrive at car:} When at the car, the operator tells the robot they have reached the car and to remember this location. E.g. ``Remember: this location is the car''. 
-\item \textbf{Handover groceries:} The operator gives the robot a command to carry something, e.g. a box or a bag.  E.g. ``Carry this box''
-  The robot puts up its arms and the operator gives the item to the robot.
-\item \textbf{Command destination:} The operator tells where the given item should go, one of the rooms in the arena. E.g. ``Bring this to the kitchen''
-\item \textbf{Delivery:} The robot then goes inside the house to deliver the item to the destination. 
-\item \textbf{Repeat until time is up:} Then, the robot goes back to the car (by itself, as it must remember where the car is) to bring another item inside. 
+  \item \textbf{Start:} The robot starts at a designated starting position in the arena, and waits for the \textit{professional} operator. The operator steps in front of the robot and tells it to follow (e.g. by saying \quotes{follow me}). The team is \emph{not} allowed to instruct the operator.
+
+  \item \textbf{Memorizing the operator:} The robot has to memorize the operator. During this phase, the robot may instruct the operator to follow a certain setup procedure.
+
+  \item \textbf{Following the operator:} When the robot signals that it is ready to start, the operator starts walking --in a natural way-- towards the car. Upon arrival, the operator will indicate the robot when they have reached their destination as instructed by the robot (e.g. by saying \quotes{here is the car} or \quotes{stop following me}).
+
+  \newcounter{enumTemp}
+  \setcounter{enumTemp}{\theenumi}
+  \item {[DSPL \& OPL]} \textbf{Bring the groceries in} \\
+  The robot is asked to deliver a bag with groceries to a specific location (e.g. \quotes{Take this bag to the kitchen table}).
```

Comment:
@kyordhel Does it mean that a robot already has a map for entire building and just does not know the car location in the map? Or we must find kitchen table by exploring environment?